### PR TITLE
Added error message to job confirm call

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by Ekman1991** [comment]
Generic error message for now. This needs further work.

- [ ] Implement correct error message (taken from the API?)

File: src/app/views/job-details/job-details.component.html [hunk 1]
```diff
@@ -33,6 +33,7 @@ <h3 class="applied-text" *ngIf="userJob && !userJob.accepted">{{'assignment.stat
       <div *ngIf="userJob && userJob.accepted && !userJob.willPerform">
         <h5 class="time-left-text">{{'assignment.status.offer' | translate}} {{userJob.remainsConfirmationHours}} {{'common.hour' | translate}} {{userJob.remainsConfirmationMinutes}} {{'common.min' | translate}}</h5>
         <div class="btn-primary-light btn-large btn provide-answer-button" (click)="onConfirmJobButtonClick()">{{'assignment.status.reply' | translate}}</div>
+        <h3 *ngIf="applyForJobErrorMessageVisible">An error occured when providing answer to this job.</h3>
```
Comment (buren) [inline src/app/views/job-details/job-details.component.html]: Move string to `src/assets/i18n/en.json` 😄

File: src/app/views/job-details/job-details.component.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export class JobDetailsComponent extends TranslationListener implements OnInit {
   private countOfApplicants: number = 0;
   private errors: any = {};
   private jobDetailsVisible: boolean;
+  private applyForJobErrorMessageVisible: boolean = false;
```
Comment (buren) [inline src/app/views/job-details/job-details.component.ts:30]: 🙇 for typing

**Review by buren — CHANGES_REQUESTED** [review]
Can't merge before hardcoded string is moved to `en.json` :)

**Review by bellevik — APPROVED** [review]

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
   "assignment.status.reply": "Provide answer",
   "assignment.status.user_company_hire": "The company wants to hire you",
   "assignment.status.you_hired": "You've been hired",
+  "assignment.status.confirmation_error" : "An error occured when providing answer to this job.",
```
Comment (buren) [inline src/assets/i18n/en.json:25]: Can't we use the error message from the API?

**Comment by buren** [comment]
A lot better than we have now (which is nothing 😉), however going forward I think we can just rely on the API to provide validations & error messages.

**Review by buren — APPROVED** [review]

File: src/assets/i18n/en.json (hunk 3 above)
Comment (buren) [reply src/assets/i18n/en.json:25]: Better than we have now, so "letting" this go...

**Pull request merged**